lib/src/sql/value/get.rs

```diff
@@ -35,10 +35,10 @@ impl Value {
 						0 => Ok(Value::Future(v.clone())),
 						// Process the future and fetch the embedded field
 						_ => {
-							v.compute(ctx, &opt.futures(true), txn, None)
-								.await?
-								.get(ctx, opt, txn, path)
-								.await
+							// Ensure futures are run
+							let opt = &opt.futures(true);
+							// Fetch the embedded field
+							v.compute(ctx, opt, txn, None).await?.get(ctx, opt, txn, path).await
```

Comment:
Thanks for clarifying the futures setting.

Also, just to be 100% clear, your change appears to enable futures for the `get` call (my change just affected the `compute` call, which is enough to fix the issue I found).

Reply:
Ahhhh yes. Hmmmm.

Reply:
Let's revert to yours!